src/DependencyInjection/Terminal42ContaoAutoRegistrationExtension.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Created by PhpStorm.
```

Comment:
Also please remove these. Ideally, we could copy the *php-cs-fixer* setup from Contao core. I can also do that later if you prefer.